website/source/docs/agent/config_entries.html.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: "docs"
+page_title: "Configuration Entry Definitions"
+sidebar_current: "docs-agent-config-entries"
+description: |-
+  Consul allows storing configuration entries centrally to be used as defaults for configuring other aspects of Consul.
+---
+
+# Configuration Entries
+
+Configuration entries can be created to provide cluster-wide defaults for various aspects of Consul. Every configuration
+entry has at least two fields: `Kind` and `Name`. Those two fields are used to uniquely identify a configuration entry.
+When put into configuration files, configuration entries can be specified as HCL or JSON objects.
+
+Example:
+
+```hcl
+Kind = "<supported kind>"
+Name = "<name of entry>"
+```
+
+Other `Kind` specific fields will be detailed below.
+
+## Usage
+
+When the agent is [configured](/docs/agent/options.html#enable_central_service_config) to enable central service configurations,
+it will look for service configuration defaults that match a registering service instance. If it finds any, the agent will merge
+those defaults with the service instance configuration. This allows for things like service protocol or proxy configuration to
+be defined globally and inherited by any affected service registrations.
+
+## Configuration Entry Kinds
+
+### Proxy Defaults - `proxy-defaults`
+
+Proxy defaults allow for configuring global config defaults across all services for Connect proxy configuration. Currently,
+only one global entry is supported.
+
+```hcl
+Kind = "proxy-defaults"
+Name = "global"
+Config {
+   proxy_specific_value = "foo"
```

Comment:
Could we have a more realistic example? Would it be possible to link to the config options for proxy defaults?

Reply:
I agree sometimes more general is better, but I think for something so new a concrete example can be more helpful.